sql.txt

```diff
@@ -1,14 +1,29 @@
 (1) Using the Customer Table, select the CustomerName, ContactName, and Country.  Use ORDER BY to order by Country.  
 Use LIMIT and OFFSET to get entries 11 through 20. Paste your SQL statement below. 
 
+SELECT CustomerName, ContactName, Country FROM Customers ORDER BY Country ASC LIMIT 10 OFFSET 10 ;
+
 (2) Select all columns from the Customer table where the ContactName starts with A.  Paste your SQL statement below.
 
+SELECT * From Customers WHERE ContactName like "A%";
+
 (3) Select all columns from the OrderDetails table where the ProductID is 51 and the quantity is greater than 10.
 Paste your SQL statement below.
 
+SELECT * from OrderDetails WHERE ProductID = 51 AND Quantity > 10;
+
 (4) Insert 3 rows into the Products table.  Note that you will have to specify a valid SupplierID and CategoryID, 
 corresponding to rows from the Supplier and Category tables.  Paste your three SQL statements below.
 
+INSERT INTO Products (ProductID, ProductName, SupplierID, CategoryID, Unit, Price) 
+   VALUES (null, "Blueberry Tea", 3, 1, "5 boxes of 10 bags", 5.50),
+          (null, "Mango Black Tea", 3, 1, "10 boxes of loose leaf", 40.00),
+          (null, "Lemon Spice", 3, 1, "2 boxes of 15 sachels", 4.65);
+
 (5) Update the two top rows of the Products Table to increase the price by 1.50. (Get SQL to do the addition for you.)  Paste your SQL statement below.
 
+UPDATE Products SET Price = (Price + 1.50) WHERE ProductID IN (1,2);
```

Comment:
Clever! I am no SQL expert, but everything here looks fine. As far as I know there isn't much of a shorthand to add to the price any differently. Instead of `IN` you can also use an `OR` statement, like `WHERE ProductID = 1 OR ProductID = 2`

Reply:
I thought there was a simpler way to do this but treehouse didn't have as many examples as I thought of.....

Feedback is wonderful!